e2e-tests/test/transactions.js

```diff
@@ -0,0 +1,76 @@
+const { assert } = require('console');
+const NodeClient = require('../node-client');
+const { nodes } = require('./../config.json');
+describe('Transactions', function () {
+    this.timeout(10_000);
+
+    let node1, node2, node3;
+    
+    /**
+     * Initialize nodes and insert Aura keys
+     */
+    before(async () => {
+        node1 = new NodeClient(nodes[0].port, nodes[0].validator);
+        node2 = new NodeClient(nodes[1].port, nodes[1].validator);
+        node3 = new NodeClient(nodes[2].port, nodes[2].validator);
+        node1.insertAuraKey(nodes[0].publicKey, nodes[0].mnemonic);
+        node2.insertAuraKey(nodes[1].publicKey, nodes[1].mnemonic);
+    });
+
+    it('Should be able to submit transaction', async function () {
+        const res = await node1.transfer(nodes[1].publicKey, 1010100101010, nodes[0].privateKey);
+        assert(!res.error, "Invalid transaction");
+        assert(res.result, "Invalid transaction");
+    });
+
+    it('Should throw on insufficient balance', async() => {
+        const res = await node1.transfer(nodes[1].publicKey, "1152921504606847000", nodes[0].privateKey);
+        assert(res.error && !res.result, "Valid transaction");
```

Comment:
The message should be `The transaction did not throw`. You can think of those error messages as strings that will be shown in the console if the error is not working correctly. In that case you want to see what was suppose to happen, what was the expected assert.